<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="clearSettingsB">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Сброс авторизации</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="smartphoneRB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Смартфон</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="tabletRB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Планшет</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="autoRB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Автоматически</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип тарифа</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bugtrackerB">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Сообщить об ошибке</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
